components/Cards/SchoolCard/SchoolCard.js

```diff
@@ -70,10 +70,20 @@ export default class SchoolCard extends Component {
 
     return (
       <Card className={styles.SchoolCard} hasAnimationOnHover={false} data-testid="SchoolCard">
-        {hasGiBill && <div className={styles.giBillRibbon}>GI Bill</div>}
+        {hasGiBill && (
+          <div className={styles.giBillRibbon} data-testid="GI Bill Ribbon">
+            GI Bill
+          </div>
+        )}
 
         <div className={styles.cardBrand}>
-          <img src={props.logoSource} alt={`${props.name} logo`} height="150" />
+          <img
+            src={props.logoSource}
+            alt={`${props.name} logo`}
+            data-testid="School logo"
+            height="150"
+          />
+          <p>{props.name}</p>
```

Comment:
So, we don't want to change the component so drastically (adding new elements) just for the sake of the test, so you should remove this line; however, your tests have highlighted an issue with our current SchoolCard implementation. Users that navigate the web with screen readers, will be left to assume the name of the Code School via the image's alt tag - that's not ideal.

Let's use the `ScreenReaderOnly` component at the top of the card to declare the name of the code school. Blind users will be happier, and our tests should be very simple.